packages/compass-sidebar/src/modules/connection-info.ts

```diff
@@ -130,16 +97,11 @@ export const changeConnectionInfo = (
   connectionInfo,
 });
 
-/**
- * Save connection info action creator.
- *
- * @param {Object} connectionInfo - The connection info object.
- *
- * @returns {Object} The action.
- */
 export const updateAndSaveConnectionInfo = (
   connectionInfo: ConnectionInfo
-): SaveConnectionInfoAction => ({
-  type: SAVE_CONNECTION_INFO,
-  connectionInfo,
```

Comment:
So similar to another place where we discussed it, you kinda want to have different action types if they are coming from different events in the application to make the logic of the app clearer (see https://redux.js.org/style-guide#model-actions-as-events-not-setters), but I don't have a strong preference for reverting it here, this store is pretty old and does a bunch of other weird things, so we can keep this

Reply:
😬 I missed taking this into account. Will raise a follow up for this.